src/main/java/org/sagebionetworks/bridge/models/studies/DemographicUser.java

```diff
@@ -0,0 +1,113 @@
+package org.sagebionetworks.bridge.models.studies;
+
+import java.util.Map;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.MapKey;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import org.sagebionetworks.bridge.models.BridgeEntity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents all demographics for a single user. Contains multiple Demographics.
+ */
+@Entity
+@Table(name = "DemographicsUsers")
+public class DemographicUser implements BridgeEntity {
+    @Nonnull
+    @Id
+    @JsonIgnore
+    private String id;
+
+    @Nonnull
+    @JsonIgnore
+    private String appId;
+
+    @Nullable
+    @JsonIgnore
+    private String studyId;
+
+    @Nonnull
+    private String userId;
+
+    // use Map for easy JSON serialization
+    @Nonnull
+    @OneToMany(mappedBy = "demographicUser", fetch = FetchType.EAGER, cascade = CascadeType.ALL, orphanRemoval = true)
+    @MapKey(name = "categoryName")
+    private Map<String, Demographic> demographics;
+
+    public DemographicUser() {
+    }
+
+    public DemographicUser(String id, String appId, String studyId, String userId,
+            Map<String, Demographic> demographics) {
+        this.id = id;
+        this.appId = appId;
+        this.studyId = studyId;
+        this.userId = userId;
+        this.demographics = demographics;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public String getAppId() {
+        return appId;
+    }
+
+    public void setAppId(String appId) {
+        this.appId = appId;
+    }
+
+    public String getStudyId() {
+        return studyId;
+    }
+
+    public void setStudyId(String studyId) {
+        this.studyId = studyId;
+    }
+
+    public String getUserId() {
+        return userId;
+    }
+
+    public void setUserId(String userId) {
+        this.userId = userId;
+    }
+
+    public Map<String, Demographic> getDemographics() {
+        return demographics;
+    }
+
+    @JsonProperty("demographics")
+    public void setDemographics(Map<String, Demographic> demographics) {
+        if (demographics != null) {
+            for (Map.Entry<String, Demographic> entry : demographics.entrySet()) {
+                if (entry.getValue() != null) {
+                    entry.getValue().setCategoryName(entry.getKey());
+                }
+            }
+        }
```

Comment:
> value has different category name than key

I added a check to make sure this is updating correctly, but just to clarify I believe at the point you've highlighted the category name is always null, so it will always be different before this method runs.